cliStocksTracker.py

```diff
@@ -26,149 +47,89 @@ def main():
     portfolio = Portfolio()
     graphs = []
 
-    # get config path
-    config_path = "config.ini"
-    portfolio_path = "portfolio.ini"
-    if args.config:
-        config_path = args.config
-    if args.portfolio_config:
-        portfolio_path = args.portfolio_config
-
-    # generate config files
-    if args.generate_config:
-        gen_config_files(config_path, portfolio_path)
-
     # read config files
-    config.read(config_path)
-    stocks_config.read(portfolio_path)
+    config.read(args.config)
+    stocks_config.read(args.portfolio_config)
 
     # verify that config files are correct
     verify_config_keys(config, stocks_config)
 
-    # get timezone for graph
-    cfg_timezone = config["General"]["timezone"]
-    if args.timezone:
-        cfg_timezone = args.timezone
-
-    # get rounding mode
-    rounding_mode = config["General"]["rounding_mode"]
-    if args.rounding_mode:
-        rounding_mode = args.rounding_mode
-
-    # get graph height/width
-    graph_width = int(config["Frame"]["width"])
-    graph_height = int(config["Frame"]["height"])
-    if args.width:
-        graph_width = args.graph_width
-    if args.height:
-        graph_height = args.graph_height
+    merge_config(config, args)
 
     portfolio.populate(stocks_config, args)
 
     portfolio.gen_graphs(
-        config["General"]["independent_graphs"] == "True" or args.independent_graphs,
-        graph_width,
-        graph_height,
-        cfg_timezone,
+        args.independent_graphs, args.graph_width, args.graph_height, args.timezone
     )
-    portfolio.print_graphs()
-    portfolio.print_table(rounding_mode)
+
+    render_engine = Renderer(args.rounding_mode, portfolio)
```

Comment:
created a separate file and class for all screen output for better manageability